eclass/aspell-dict-r1.eclass

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+# @ECLASS: aspell-dict-r1.eclass
+# @MAINTAINER:
+# maintainer-needed@gentoo.org
+# @AUTHOR:
+# Original author: Seemant Kulleen
+# @BLURB: An eclass to streamline the construction of ebuilds for new aspell dicts
+# @DESCRIPTION:
+# The aspell-dict-r1 eclass is designed to streamline the construction of
+# ebuilds for the new aspell dictionaries (from gnu.org) which support
+# aspell-0.50. Support for aspell-0.60 has been added by Sergey Ulanov.
+
+# @ECLASS-VARIABLE: ASPELL_LANG
+# @REQUIRED
+# @DESCRIPTION:
+# Which language is the dictionary for? It's used for the DESCRIPTION of the
+# package.
+
+# @ECLASS-VARIABLE: ASPOSTFIX
+# @REQUIRED
+# @DESCRIPTION:
+# What major version of aspell is this dictionary for?
+
+case "${EAPI:-0}" in
+	[0-5])
+		die "aspell-dict-r1.eclass is banned in EAPI ${EAPI:-0}"
+		;;
+	6)
+		;;
+	*)
+		die "Unknown EAPI ${EAPI:-0}"
+		;;
+esac
+
+EXPORT_FUNCTIONS src_configure src_install
+
+if [[ ! ${_ASPELL_DICT_R1} ]]; then
+
+MY_P=${P%.*}-${PV##*.}
```

Comment:
Would be helpful to use versionator.eclass here or leave a comment on what this does exactly. Otherwise, it's an imminent pitfall. Plus, declaring `MY_P` in eclass is a bad idea, better namespace it.